python/pysnaptest/snapshot.py

```diff
@@ -122,7 +194,11 @@ def snapshot(func: Callable) -> Callable: ...
 
 @overload
 def snapshot(
-    *, filename: Optional[str] = None, folder_path: Optional[str] = None
+    *,
+    filename: Optional[str] = None,
+    folder_path: Optional[str] = None,
+    redactions: Optional[Dict[str, str]] = None,
+    snapshot_format: str = "csv",
```

Comment:
Should we name this argument with something that specifically calls out this only affects data frames at the moment?